infrastructure/images/moduletest-reset-jobs/templates/moduletest-file-reset-job.template.yaml

```diff
@@ -0,0 +1,36 @@
+# This manifest will fire up a customer Kubernetes Job in moduletest namepaces.
+# It will synchronize all files from the corresponding main environment to the moduletest environment
+
+
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: moduletest-reset
+
+---
+
+{{- range .Values.projects }}
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: file-reset
+  labels:
+    app: moduletest-file-reset-job
+spec:
+  ttlSecondsAfterFinished: 36000 # 10 hours leaving time to investigate failed jobs
+  backoffLimit: 3
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: file-reset-container
+          image: registry.local.deranged.dk/file-reset:dev
+          # image: ghcr.io/danskernesdigitalebibliotek/dpl-platform/lagoon-redeploy-controller:lrc-1.0.1-rc
+          command:
+            - /bin/bash
+            - -c
+            - ./moduletest-files-reset-job.mjs {{ . }}
+            
+---
+{{- end }}
```

Comment:
This should run on the general nodes, right? You'll need some node tolerations and affinities.

Reply:
I haven't actually given any thought to to that. What would be your suggestion? @ath88

Reply:
What are your considerations on this? What nodes would you consider running this on?

Reply:
fixed: https://github.com/danskernesdigitalebibliotek/dpl-platform/pull/634/commits/e2bee832a565e06cb44827041b4730709ac193ef

Reply:
Looks better here https://github.com/danskernesdigitalebibliotek/dpl-platform/commit/45740628fbefab3b2dec1e2102c64caa2bd74a09 - remember to put the SHA in. Makes it quicker for me to review. :)